test/content/support.js

```diff
@@ -241,4 +241,17 @@ function resetDB() {
 	}).then(function() {
 		return Zotero.Schema.schemaUpdatePromise;
 	});
-}
\ No newline at end of file
+}
+
+
+/**
+ * Imports an attachment from a test file.
+ * @param {string} filename - The filename to import (in data directory)
+ * @returns {Promise} as returned by Zotero.Attachments.importFromFile
+ */
+function importFromTestFile(filename) {
+	let testfile = getTestDataDirectory();
+	testfile.append(filename);
```

Comment:
I think subdirectories will come in handy for test data files, so this should take a relative path (e.g., "search/foo.html").  (Just splitting on forward slashes would be fine.) Then we can move and rename the HTML files.